web/api/badges.js

```diff
@@ -79,6 +89,12 @@ exports.register = function (server, eOptions, next) {
       auth: auth.apiUser,
       description: 'Request a badge',
       tags: ['api', 'users'],
+      validate: {
```

Comment:
Isn't this actually an endpoint that can just be removed? Badges are auto-awarded for some time now. No need to claim them.